examples/main.tf

```diff
@@ -6,16 +6,16 @@ terraform {
       source = "instaclustr/instaclustr"
       //Change the source as per below to work with a local development copy on terraform version >=13
       //source = "terraform.instaclustr.com/instaclustr/instaclustr"
-      version = ">= 1.0.0"
+      version = ">= 1.14.8"
     }
   }
 }
 
 provider "instaclustr" {
-  username = "<Your instaclustr username here>"
-  api_key = "<Your provisioning API key here>"
+  username = ""
+  api_key = ""
```

Comment:
You should revert these changes here, keep the `<Your instaclustr username here>` prefills